apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -123,14 +108,19 @@ public static void setFileAttachment(Uri fileAttachment) {
         sCrashesListener.setFileAttachment(fileAttachment);
     }
 
-    native void setupNativeCrashesListener(String path);
-
     static void startAppCenter(Application application, String startTypeString) {
-        int position = MainActivity.sSharedPreferences.getInt(LATENCY_SECONDS_KEY, 0);
-        long latency = sLatencyValues[position];
-        try{
+        int latency = DEFAULT_TRANSMISSION_INTERVAL_IN_SECONDS;
+        if (MainActivity.sSharedPreferences.contains(ANALYTICS_TRANSMISSION_INTERVAL_KEY)) {
```

Comment:
You can simplify that by just doing MainActivity.sSharedPreferences.getInt(key, defaultValue) instead